akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -228,6 +234,10 @@ object Behavior {
     override def toString = "Same"
   }
 
+  private[akka] class FailedBehavior(val cause: Throwable) extends Behavior[Nothing] {
+    override def toString: String = s"Failed(${cause.getMessage})"
```

Comment:
I think `toString` of the the cause is typically better, e.g. NullPointerException